exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function here
-
+function createLongGreeting(name, age) {
+  return `Hello, my name is ${name} + and I'm ${age} years old`;
```

Comment:
Are you using string interpolation? If so, maybe we don't need the plus sign?

Reply:
Yes, sure I made the change 😊